styles.css

```
@@ -122,18 +136,70 @@ textarea {
+    transition: color 160ms ease;
+}
+
+.nav-links a::after,
+.footer nav a::after,
+.page-footer nav a::after {
+    position: absolute;
+    right: 0;
+    bottom: -6px;
+    left: 0;
+    height: 1px;
+    background: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)